src/compiler.js

```diff
@@ -76,114 +76,115 @@ const createRegularCellDefintion = cell => {
     cellReferences
   };
 };
+const cellPromise = async (cell, main, observer, resolveModule) => {
```

Comment:
Good call abstracting this out: this will make `Compiler.cell` much easier to implement